python/cudf/cudf/core/column/decimal.py

```diff
@@ -96,7 +94,12 @@ def _binaryop(self, other: ColumnBinaryOperand, op: str):
                 "__le__",
                 "__ge__",
             }:
-                result = libcudf.binaryop.binaryop(self, other, op, bool)
+                result = libcudf.binaryop.binaryop(lhs, rhs, op, bool)
+            else:
+                raise NotImplementedError(
+                    f"{op} not supported for types following dtypes:"
```

Comment:
Needs a space. But what does “types following dtypes” mean? Can we just say “dtypes”?
```suggestion
                    f"{op} not supported for the following dtypes: "
```

Reply:
The intention was to show dtypes to the user rather than leaving then confused about which op isn't supported between which two types.